collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -511,4 +512,10 @@ public ProjectsRepository providesProjectsRepository(UUIDGenerator uuidGenerator
     public UUIDGenerator providesUUIDGenerator() {
         return new UUIDGenerator();
     }
+
+    @Provides
+    @Singleton
```

Comment:
This doesn't need to be a singleton right? Are you just thinking it is safe to be one and so it keeps the object count down?

Reply:
Right, there is no need to add singleton annotation there, Fixed.